tests/dialects/test_bigquery.py

```diff
@@ -1090,6 +1090,13 @@ def test_bigquery(self):
             },
             pretty=True,
         )
+        self.validate_all(
+            "SELECT x % 10",
+            write={
```

Comment:
you could do read={
  postgres: "select x % 10" to test this
}

Reply:
I get `SELECT MOD(x, 10)` in postgres now but that is valid: https://www.postgresqltutorial.com/postgresql-math-functions/postgresql-mod/

Reply:
it's being parsed as anonymous, you need to add a parser

FUNCTIONs = { "MOD": ...}

Reply:
Updated: https://github.com/tobymao/sqlglot/pull/3157/commits/8c161bb7547e458f0c94898f78dbb9fdbf3470b4